docs/DeveloperGuide.md

```diff
@@ -348,6 +348,30 @@ Throughout the onboarding guide, Overlays and Highlights are used to direct the
 ##### The `Overlay` Class
 The Overlay class is implemented using 2 translucent panes binded to the top and bottom of the OnboardingWindow. This makes it possible to create an desired area of focus by leaving only an area uncovered.
 
+### Markdown-like Text Processing
+We built a text processor that processes common text modifications into UI elements. This is mainly used in the `ResultDisplay` field, where it shows text of a `CommandResult` each time a command is executed. In particular, it is used in the error messages, in order to present error messages better,
+
+Example of styled text:
+![StyledTextExample](images/StyledTextExample.png)
+
+It can be easily used in other components of ContaX if the need arises.
+
+It currently supports:
+- Bold
+  - Enclosing text with `*`
+- Italics
+  - Enclosing text with `**`
```

Comment:
Isn't italics and bold reversed here? ** is bold and * is italics.

Reply:
thanks for the catch, fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/251/commits/db4e377fa83fcfca66d948db87fb4374d8fd5961